### ISSUE TITLE
User should not be able to blank work by other mappers, only replace

### ISSUE THREAD

**Issue by matkoniecz** [colouring-cities/colouring-london#240]
Expected: user edits objects (for example "land use"), removes building classification, leaves it blank, saves - and edit is refused

What happens: edit is not refused

It likely is implemented but disabled/reverted and that was intended to be temporary (per @polly64)

**Comment by mdsimpson42** [comment]
This is about users blanking other users data, not undoing their own changes.
I would expect an undo feature to only be available to the user who made the changes (i.e. you can't undo something someone else has done).

**Comment by polly64** [comment]
@mdsimpson42 yes this was exactly the issue- i.e. making you overwrite rather than delete - can you not undo other people's edits? maybe ones I have tested it on are my own which is why I can undo them and leave blank

**Comment by polly64** [comment]
@mdsimpson42 ok great yes. Is this process active or do we need to do something? Do close if complete
